bin/test/animation_assets/test_manifest_builder.rb

```diff
@@ -10,7 +10,7 @@ def setup
   def test_get_animation_strings_for_spritelab_quietly
     animation_strings = nil
 
-    VCR.use_cassette('animations/manifest_spritelab_strings') do
+    VCR.use_cassette('animations/manifest_spritelab_strings', record: :none) do
```

Comment:
why is this change necessary?

Reply:
The vcs cassette for this test was created manually to cover all the code conditions at once, so it shouldn't be ever rewritten by the real response